File: android/app/src/main/kotlin/com/anhquan/unisync/database/entity/PairedDeviceEntity.kt
package com.anhquan.unisync.database.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "paired_devices")
data class  PairedDeviceEntity(
    @PrimaryKey val id: String,
    val name: String,
    val type: String
)

File: lib/database/entity/paired_device.entity.dart
import 'package:floor/floor.dart';

import '../tables.dart';

@Entity(tableName: DBTables.PAIRED_DEVICES)
class PairedDeviceEntity {
  PairedDeviceEntity({
    required this.id,
    required this.name,
    required this.type,
    this.unpaired = 0,
  }) {
    lastAccessed = DateTime.now().millisecondsSinceEpoch;
  }

  @primaryKey
  final String id;
  final String name;
  final String type;
  late final int lastAccessed;
  final int unpaired;
}
